var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -282,16 +293,19 @@ class Mapl(CMakePackage):
     depends_on("pflogger@1.9.1: +mpi", when="@2.23:2.39+pflogger")
     depends_on("pflogger@1.9.5: +mpi", when="@2.40:2.43+pflogger")
     depends_on("pflogger@1.11.0: +mpi", when="@2.44+pflogger")
-    depends_on("pflogger@1.14.0: +mpi", when="@2.45:+pflogger")
+    depends_on("pflogger@1.14.0: +mpi", when="@2.45:2.47+pflogger")
```

Comment:
All other dependencies are order from newest to oldest, except pflogger

Reply:
Will fix up.